scss/modules/_datatables.scss

```diff
@@ -253,3 +278,42 @@
     @include cell-width($width);
   }
 }
+
+@include media($med) {
+  .column--state {
+    width: 50px;
+  }
+
+  .column--small {
+    width: 90px;
+  }
+
+  .column--med {
+    width: 15%;
+  }
+
+  .column--large {
+    width: 20%;
+  }
+
+  .column--xl {
+    width: 30%;
+  }
+
+  .panel-active {
+    .column--xl,
+    .column--large {
+      width: 50%;
+    }
+
+    .column--med {
+      width: 25%;
+    }
+  }
+}
```

Comment:
Hmm, good call on the column sizing. I had added the `.cell--5` classes, but to have a standard set seems better.